javascript/README.md

```diff
@@ -643,3 +651,101 @@ const { left, top } = processInput(input);
 ```
 
 [⇧ top](#javascript-guide)
+
+---
+
+## Strings
+
+### 5.1 Quotes
+
+Use single quotes `''` for strings.
+
+#### Examples
+
+🚫 Nope. 🚫
+
+```js
+const name = "Cloud Four";
+
+// Template literals should contain interpolation or newlines.
+const name = `Cloud Four`;
+```
+
+🎉 Yep! 🎉
+
+```js
+const name = 'Cloud Four';
+```
+
+#### Resources
+
+- ESLint: [quotes]
+
+### 5.2 Template Literals
+
+When programmatically building up strings, use [template literals][Template Literals] instead of concatenation.
+
+> Why? Template literals (template strings) give you a readable, concise syntax with proper newlines and string interpolation features.
+
+#### Examples
+
+🚫 Nope. 🚫
+
+```js
+function sayHi(name) {
+  return 'How are you, ' + name + '?';
+}
+
+function sayHi(name) {
+  return ['How are you, ', name, '?'].join();
+}
+```
+
+🎉 Yep! 🎉
+
+```js
+function sayHi(name) {
+  return `How are you, ${name}?`;
+}
+```
+
+#### Resources
+
+- ESLint:
+  - [prefer-template]
+  - [template-curly-spacing]
+
```

Comment:
I don't think we need the link to `template-curly-spacing` here. It is stylistic, and Prettier handles that